actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -45,12 +45,22 @@ def action
       end
 
       def internal?
-        controller.to_s =~ %r{\Arails/(info|mailers|welcome)} || path =~ %r{\A#{Rails.application.config.assets.prefix}\z}
+        internal_controller? || internal_asset?
       end
 
       def engine?
         rack_app.respond_to?(:routes)
       end
+
+      private
+        def internal_controller?
+          controller.to_s =~ %r{\arails/(info|mailers|welcome)}
```

Comment:
Shouldn't this be a escaped, capital A, like this: `\A`? 

Also, why use the `%r{}` syntax, as opposed to `//`?